modules/ROOT/pages/cassandra/cassandra-xml-maven.adoc

```diff
@@ -0,0 +1,59 @@
+= XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+XML and POM file configuration information and XML-related usage topics.
+
+== Add a Namespace for the Connector
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/cassandra-db
+http://www.mulesoft.org/schema/mule/cassandra-db/current/mule-cassandra-db.xsd
+----
+
+== Configure the Maven POM File
+When creating an app manually from the XML, set up the pom.xml for your project:
+
+ [source,xml,linenums]
```

Comment:
This is rendering as part of the code--I think if you remove the space it'll work? Same for line 34.